src/ActionCreators/AuthActionCreators.js

```diff
@@ -13,4 +11,16 @@ const LoginFailed = errorMessage => ({
   payload: { errorMessage },
 });
 
-export { AUTHENTICATED, AUTHENTICATION_FAILED, Login, LoginFailed };
+const isAuthenticated = () => {
+  let payload = false;
+  const token = localStorage.getItem('openmrs-token');
```

Comment:
Just wonder why you name your token openmrs-token, did you look at the name used when setting it in the localstorage

Reply:
LGTM